downstream/modules/aap-hardening/proc-install-user-pki.adoc

```diff
@@ -13,29 +13,39 @@ Use the following inventory variables to configure the infrastructure components
+| `web_server_ssl_key` | `controller_tls_key` | The file name of the {ControllerName} PKI key located in the installer directory.
 
-| `automationhub_ssl_key` | The file name of the {PrivateHubName} PKI key located in the installer directory.
+| `automationhub_ssl_cert` | `hub_tls_cert` | The file name of the {PrivateHubName} PKI certificate located in the installer directory.
 
-| `postgres_ssl_cert` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
+| `automationhub_ssl_key` | `hub_tls_key` | The file name of the {PrivateHubName} PKI key located in the installer directory.
 
-| `postgres_ssl_key` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
+| `postgres_ssl_cert` | `postgresql_tls_cert` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
 
-| `automationedacontroller_ssl_cert` | The file name of the {EDAcontroller} PKI certificate located in the installer directory.
+| `postgres_ssl_key` | `postgresql_tls_key` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
 
-| `automationedacontroller_ssl_key` | The file name of the {EDAcontroller} PKI key located in the installer directory.
+| `automationedacontroller_ssl_cert` | `eda_tls_cert` | The file name of the {EDAcontroller} PKI certificate located in the installer directory.
+
+| `automationedacontroller_ssl_key` | `eda_tls_key` | The file name of the {EDAcontroller} PKI key located in the installer directory.
 |===
 
-When multiple {ControllerName} are deployed with a load balancer, the `web_server_ssl_cert` and `web_server_ssl_key` are shared by each controller. To prevent hostname mismatches, the certificate's Common Name (CN) must match the DNS FQDN used by the load balancer. This also applies when deploying multiple {PrivateHubName} and the `automationhub_ssl_cert` and `automationhub_ssl_key` variables. If your organizational policies require unique certificates for each service, each certificate requires a Subject Alt Name (SAN) that matches the DNS FQDN used for the load-balanced service. To install unique certificates and keys on each {ControllerName}, the certificate and key variables in the installation inventory file must be defined as per-host variables instead of in the `[all:vars]` section. For example:
+When multiple {ControllerName} are deployed with a load balancer, the `web_server_ssl_cert` and `web_server_ssl_key` are shared by each {ControllerName}. 
```

Comment:
{ControllerName} should be "{Gateway}s", and the variables should be `gateway_tls_cert` and `gateway_tls_key`